pages/index.tsx

```diff
@@ -92,11 +92,11 @@ const Home = ({ stars }: Props) => {
 };
 
 Home.getInitialProps = async () => {
-	const res = await fetch("https://api.github.com/repos/zeit/next.js");
-	const json = await res.json();
+	// const res = await fetch("https://api.github.com/repos/zeit/next.js");
+	// const json = await res.json();
 	return {
 		namespacesRequired: includeDefaultNamespaces(["common"]),
-		stars: json.stargazers_count,
+		// stars: json.stargazers_count,
```

Comment:
Having this removed will break the template that uses the `stars` attribute. Either leave the API call, or remove the attribute from the component. I'd be inclined to leave the API call for reference purposes.

Reply:
Okay, re-added! Commented out just because it crashes program on prem